src/webpack.config.ts

```diff
@@ -18,6 +18,7 @@ const packagePath = isCLI ? '.' : '@dojo/cli-build-webpack';
 const CoreLoadPlugin = require(`${packagePath}/plugins/CoreLoadPlugin`).default;
 const I18nPlugin = require(`${packagePath}/plugins/I18nPlugin`).default;
 const basePath = process.cwd();
+const bannerPath = isCLI ? path.join(__dirname, './banner.md') : require.resolve(`${packagePath}/banner.md`);
```

Comment:
can this not just be `packagePath`? which is adjusted accordingly based on `isCLI` anyway?